apiserver/facade/interface.go

```diff
@@ -200,6 +200,10 @@ type Authorizer interface {
 	// target by the authenticated entity.
 	HasPermission(operation permission.Access, target names.Tag) (bool, error)
 
+	// EntityHasPermission reports whether the given access is allowed for the given
+	// target by the given entity.
+	EntityHasPermission(entity names.Tag, operation permission.Access, target names.Tag) (bool, error)
```

Comment:
I feel like we've got the abstractions all wrong if we need to surface what is essentially a finer-grained `HasPermission` in this public interface.